src/observability/grafana/grafana_launcher.star

```diff
@@ -118,14 +118,42 @@ def provision_dashboards(plan, service_url, dashboard_sources):
     if len(dashboard_sources) == 0:
         return
 
+    def grr_compat(plan, files, name):
+        """
+        Checks if the dashboards are already compatible with grizzly, by checking if the `apiVersion` is set within all
+        of the dashboard json files.
+        """
+        result = plan.run_sh(
+            description="Checking for grizzly compatibility",
+            run="find {0}/dashboards -name '*.json' -exec grep -i 'grizzly.grafana.com' {{}} \\+".format(name),
+            acceptable_codes=[0, 1],
+            files=files
+        )
+
+        return result.code == "0"
```

Comment:
This won't work because `code` is a [future reference](https://docs.kurtosis.com/advanced-concepts/future-references/) 😢